src/libs/chartkit-error/chartkit-error.ts

```diff
@@ -12,7 +12,7 @@ export const CHARTKIT_ERROR_CODE = {
 
 export class ChartKitError extends Error {
     readonly code: number | string;
-    readonly isChartKitError = true;
+    readonly isCustomError = true;
```

Comment:
To easier compatibility with old code in DL. It will be changed to `isChartKitError ` later